main/src/library/Iterable.flix

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright 2021 Nina Andrup Pedersen
```

Comment:
Its 2022, but I will let you fix that later :)

Reply:
Haha, I will have to get used to it!